certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/validation/model/ValidationUtil.java

```diff
@@ -0,0 +1,26 @@
+package se.inera.intyg.certificateservice.domain.validation.model;
+
+import java.util.Optional;
+import se.inera.intyg.certificateservice.domain.certificate.model.ElementData;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementId;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.FieldId;
+
+public class ValidationUtil {
+
+  private ValidationUtil() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static ValidationError errorMessage(
```

Comment:
Not sure if the util adds much value. All arguments are anyway passed. If you really believe its value, then add the method instead as a factory method to the ValidationError class and remove this.